csm/conf/prepare.py

```diff
@@ -127,10 +145,11 @@ def _set_db_host_addr(self):
         :param backend: Databased backend. Supports Elasticsearch('es'), Consul ('consul').
```

Comment:
Backend is not the part params .. remove it from  method doc..  Write proper description.

Reply:
Updated required changes in adding proper description for method.